firmware/ventilator-controller-stm32/Core/Test/Inc/Pufferfish/Test/Util.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Util.h
+ *
+ *  Created on: Nov 17, 2020
+ *      Author: Renji Panicker
+ *
+ *  A set of helper functins for writing and debugging test cases
```

Comment:
Since the functions in this file are also used to help construct examples of inputs and expected outputs, does this file also need to have unit tests? e.g. so that we can be more confident in the correctness of `convertByteVectorToHexString`.